.github/workflows/main.yml

```diff
@@ -43,7 +43,18 @@ jobs:
         # If clang, use the default version. Otherwise the compiler install with apt-get.
         [ "${{ matrix.compiler }}" != "clang" ] && sudo apt-get install ${{ matrix.compiler }}
 
-        sudo apt-get install libbz2-dev libhts-dev libjemalloc-dev libboost-all-dev
+        sudo apt-get --allow-downgrades install \
+            g++-8=8-20180414-1ubuntu2 \
+            gcc-8-base=8-20180414-1ubuntu2 \
+            gcc-8=8-20180414-1ubuntu2 \
+            libstdc++-8-dev=8-20180414-1ubuntu2 \
+            cpp-8=8-20180414-1ubuntu2 \
+            libgcc-8-dev=8-20180414-1ubuntu2 \
+            libmpx2=8-20180414-1ubuntu2 \
+            libbz2-dev \
+            libhts-dev \
+            libjemalloc-dev \
+            libboost-all-dev
```

Comment:
Can you add a special build with this version of gcc?
The existing gcc-8 will still use 8.4.0.

Reply:
so have both gcc-8.4 and 8.0? because currently it's using 8.0
```
-- The C compiler identification is GNU 8.0.1
-- The CXX compiler identification is GNU 8.0.1
```

Reply:
True. As you wish then, but maybe add a check and install all these only if it's the gcc-8 build.

Reply:
ok. The gcc-9 and gcc-10 builds still use the correct version, but it's just a waste of time downgrading.

Reply:
done